src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -165,6 +165,8 @@ public static void TestDispose()
         }
 
         [Fact]
+        // Crashing on macOS 10.13 Beta
```

Comment:
if we start doing this more, we can add a version detection method to PlatformDetectin.